Monika After Story/game/zz_backup.rpy

```diff
@@ -385,6 +385,7 @@ label mas_backups_you_have_corrupted_persistent:
         show chibika at sticker_move_n
         "I was unable to find a working backup persistent."
 
+        "Do you have your own backups?"
```

Comment:
Can you actually mark this area with a TODO for me to decide if text speed should be disabled/forced here?